### ISSUE TITLE
Not compatible with Proxmox 7.3-3 LXC

### ISSUE THREAD

**Issue by 60753b9e** [blesta/module-proxmox#18]
# Describe
I can't create an LXC server through this plugin, because the error flag in the code **"proxmox/apis/commands/proxmox_vserver.php" line 72**, openvz should be lxc, even if it is modified here, an error will still be thrown.

# Environment 
Blesta Version: Blesta 5.5.3
Proxmox Version: 7.3-3
OS Version: Arch Linux x86
PHP Version: 7.4..33

# Additional code
Before
```
xxxx.com
{"data":null}
```
After
```
xxxx.com
{"data":null,"errors":{"ip_address":"property is not defined in schema and the schema does not allow additional properties","cpus":"property is not defined in schema and the schema does not allow additional properties","disk":"property is not defined in schema and the schema does not allow additional properties"}}
```

**Comment by TeYroXOfficial** [comment]
Current version of proxmox doesn't have support for lxc, I send pull request many months ago but still nothing.
You can use my fork if you want to use LXC 
https://github.com/TeYroXOfficial/module-proxmox

**Comment by JReissmueller** [comment]
This was resolved by #20 and improved by a few other PRs
